.rails-lens.yml

```diff
@@ -0,0 +1,8 @@
+# .rails-lens.yml (optional)
+adapter: auto  # auto-detects PostgreSQL/MySQL/SQLite3
+marker_format: rails-lens:schema  # LLM-friendly markers
+include_notes: true  # Performance recommendations
+extensions:
+  enabled: true  # ClosureTree, PostGIS, etc.
```

Comment:
While you don't use any of the extensions currently, you can build you own for your business requirements.

For now ClosureTree and PostGIS are embedded, but they will move to their respective gem before we hit 1.0.0.


Note: Fixture and Fabricators don't get annotated for now.